main.go

```diff
@@ -70,19 +70,25 @@ func runElectron() {
 	if len(electronPath) == 0 {
 		electronPath = config.ConfigDir + "/electron"
 	}
-	var a, err = astilectron.New(l, astilectron.Options{
+
+	path, err := os.Getwd()
```

Comment:
There are some configuration value which I thought could be used instead - what is the purpose of the following:

* `config.HomeDir`
* `config.DataDir`

Perhaps even `config.AxolotlWebDir` could be put to use?